### DISCUSSION TITLE
cant import plugins

### DISCUSSION THREAD

**Discussion by c0llin77** [runelite/runelite#17081] [Q&A]
hello i play on windows pc/ macbook and recently had to send macbook in for repair, they wiped everything and when i logged into windows under my login email, all my plugins are gone? it looks like they are saved but i cant import or use cloud thing. help please.

**Comment by carlossclaro** [comment]
Could you provide the logs and also an image of what you are seeing on your end? Were you previously logged in before you sent in your Macbook for a repair?
